src/config/i18n/index.js

```diff
@@ -8,30 +8,29 @@ import LanguageDetector from 'i18next-browser-languagedetector';
 // See https://www.i18next.com/how-to/add-or-load-translations
 
 import defaults from './defaults.json';
-import en from './en/resource.json';
 import es from './es/resource.json';
-import pt from './pt-br/resource.json';
-
-const resources = {
-  defaults: { translation: defaults },
-  en: { translation: en },
-  es: { translation: es },
-  pt: { translation: pt }
-};
+import fr from './fr/resource.json';
+import ptBR from './pt-br/resource.json';
 
 i18n
   .use(LanguageDetector)
   .use(initReactI18next)
   .init({
-    resources,
-
-    // do not load a fallback
-    fallbackLng: ['en', 'defaults'],
-
+    detection: {
+      order: ['querystring', 'htmlTag']
```

Comment:
Noticed default settings we're letting browser override the RP setting. Query string just here for testing so you can add `lng` to the URL to try a different language